configs/ilql_summarize_t5.yml

```diff
@@ -0,0 +1,52 @@
+train:
+  seq_length: 550
+  epochs: 100
+  total_steps: 10000
+  batch_size: 8
+
+  checkpoint_interval: 100000
+  eval_interval: 500
+
+  pipeline: "PromptPipeline"
+  trainer: "AccelerateILQLTrainer"
+  checkpoint_dir: "gpt-j-ilql-summarize-duy"
+  #  tracker: null
+
+model:
+  model_path: "/admin/home-duyphung/software/github/pvd/trlx/examples/summarize_rlhf/flan-t5-qa/checkpoint-3000"
```

Comment:
this path is unfortunately private, but you don't have to use sft and can just start from hf's checkpoint here

Reply:
Yes I will upload T5 SFT for to hf.